### ISSUE TITLE
No module named 'undetected_chromedriver.v2'

### ISSUE THREAD

**Issue by QazCetelic** [You-OpenSource/You-Python#47]
## 🐛 Bug Report

```
Traceback (most recent call last):
  File "/home/username/.local/bin/youdotcom", line 5, in <module>
    from youdotcom.__main__ import app
  File "/home/username/.local/lib/python3.11/site-packages/youdotcom/__init__.py", line 13, in <module>
    from .youchat import Chat
  File "/home/username/.local/lib/python3.11/site-packages/youdotcom/youchat.py", line 13, in <module>
    import undetected_chromedriver.v2 as uc
ModuleNotFoundError: No module named 'undetected_chromedriver.v2'
```

## 🔬 How To Reproduce

1. Interact with package in any way (such as running `youdotcom` in the terminal)

### Code sample

```shell
youdotcom
```

### Environment

* OS: Linux 
* Python version: 3.11.1

**Comment by SilkePilon** [comment]
Hi there thx for the bug report! Im working on it rn. It seems to be an update to the library im using. If you need it fixed now you can change the __init__.py file to remove the .v2

**Comment by SilkePilon** [comment]
Hope this helps.

**Comment by SilkePilon** [comment]
this is now fixed in 1.2.1!

sorry for the late fix
